### PR TITLE
add information about the pastebin instance.

### PR DESCRIPTION
**Pull request by lxea**

### CODE REVIEW

File: docs/services/paste.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Redbrick uses [hastebin](https://github.com/seejohnrun/haste-server) for pastes
+
+## Hastebin
+
+The hastebin instance is run in docker on zeus.
+
+The `Dockerfile` is as follows
+
+```Dockerfile
+FROM node:boron
```
Comment (butlerx) [inline docs/services/paste.md]: Could we use boron-alpine?

File: docs/services/paste.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+the `docker-compose.yml` is
+
+```yaml
+version: '2'
+services:
+  redis:
+    image: "redis:alpine"
+  haste:
+    build: .
+    ports:
```
Comment (butlerx) [inline docs/services/paste.md]: Could we give the image a name?

**Review by butlerx — CHANGES_REQUESTED** [review]
Just things to do with docker in general

File: mkdocs.yml [hunk 3]
```diff
@@ -28,6 +28,7 @@ pages:
     - News: services/news.md
     - Ossec: services/ossec.md
     - Postgres: services/postgres.md
+    - Pastebin: services/paste.md
```
Comment (butlerx) [inline mkdocs.yml]: Paste should be before postgres

File: docs/services/paste.md (hunk 2 above)
Comment (lxea) [reply docs/services/paste.md]: the redis image?

Comment (butlerx) [reply docs/services/paste.md]: no the paste container

Comment (butlerx) [reply docs/services/paste.md]: just something like `image: redbrick/hastebin`

**Review by butlerx — APPROVED** [review]

**Pull request merged**